csrankings-l.csv

```diff
@@ -236,6 +236,7 @@ Lei Guo 0002,NWPU,https://teacher.nwpu.edu.cn/en/guolei.html,NOSCHOLARPAGE
 Lei Jiang 0001,Indiana University,http://homes.sice.indiana.edu/jiang60,1sXorAAAAAJ
 Lei Jimmy Ba,University of Toronto,https://jimmylba.github.io,ymzxRhAAAAAJ
 Lei Ju,Shandong University,https://faculty.sdu.edu.cn/julei/en/index.htm,7DZXXBQAAAAJ
+Lei Wang,University of Wollongong,https://scholars.uow.edu.au/lei-wang,5ClujcoAAAAJ

```

Comment:
Same issue here; add 0001. See https://dblp.org/pid/w/LeiWang1.html.

Reply:
Same issue for another faculty member (Ping Yu), who needs a 0004 suffix. This was in the instructions, by the way. https://dblp.org/pid/72/1358-4.html

Reply:
thanks, I fixed